src/pages/organizers/[organizerId]/preview/OrganizerLabels.tsx

```diff
@@ -0,0 +1,154 @@
+import { Alert } from '@/ui/Alert';
+import { FormElement } from '@/ui/FormElement';
+import { Icon, Icons } from '@/ui/Icon';
+import { Inline } from '@/ui/Inline';
+import { Stack } from '@/ui/Stack';
+import { Text, TextVariants } from '@/ui/Text';
+import { getGlobalBorderRadius, getValueFromTheme } from '@/ui/theme';
+import { Typeahead } from '@/ui/Typeahead';
+import { getUniqueLabels } from '@/utils/getUniqueLabels';
+
+type OrganizerLabelProps = {
+  organizer: Organizer;
+};
+
+export const OrganizerLabels = ({ organizer }: OrganizerLabelProps) => {
+  const { t } = useTranslation();
+  const ref = useRef(null);
+  const router = useRouter();
+  const [query, setQuery] = useState('');
+  // @ts-expect-error
+  const labelsQuery: UseQueryResult<{ member: Label[] }> = useGetLabelsByQuery({
+    query,
+  });
+
+  const scope = ScopeTypes.ORGANIZERS;
+  const organizerId = useMemo(
+    () => router.query.organizerId as string,
+    [router.query.organizerId],
+  );
+
+  const options = labelsQuery.data?.member ?? [];
+  const [labels, setLabels] = useState<string[]>(
+    getUniqueLabels(organizer) ?? [],
+  );
+  const addLabelMutation = useAddOfferLabelMutation();
+  const removeLabelMutation = useRemoveOfferLabelMutation();
+  const getButtonValue = getValueFromTheme('button');
+
+  const isWriting = addLabelMutation.isLoading || removeLabelMutation.isLoading;
+  const [isInvalid, setIsInvalid] = useState(false);
+  return (
+    <Inline flex={1} spacing={5}>
+      <Stack flex={1} opacity={isWriting ? 0.5 : 1} spacing={2}>
+        <FormElement
+          id={'labels-picker'}
+          label={
+            <Text
+              fontSize="0.8rem"
+              fontWeight="normal"
+              variant={TextVariants.MUTED}
+            >
+              {t('create.additionalInformation.labels.info')}
+            </Text>
+          }
+          loading={isWriting}
+          error={
+            isInvalid
+              ? t('create.additionalInformation.labels.error')
+              : undefined
+          }
+          Component={
+            <Typeahead
+              ref={ref}
+              name={'labels'}
+              isInvalid={isInvalid}
+              isLoading={labelsQuery.isLoading}
+              options={options}
+              labelKey={'name'}
+              allowNew
+              newSelectionPrefix={t(
+                'create.additionalInformation.labels.add_new_label',
+              )}
+              onSearch={setQuery}
+              onChange={async (newLabels: Label[]) => {
+                const label = newLabels[0]?.name;
+                if (!label || !label.match(LABEL_PATTERN)) {
+                  return setIsInvalid(true);
+                }
+
+                setIsInvalid(false);
+                await addLabelMutation.mutateAsync({
+                  id: organizerId,
+                  scope,
+                  label,
+                });
+
+                setLabels(uniq([...labels, label]));
```

Comment:
Instead of using lodash uniq you can also do it a native js way. 

```ts
const labels = [1, 2, 3, 4]
const label = 3

// inline
[...new Set([...labels, label])]

// as a util function with correct type inference
const uniqueValues = <T,>(...values: T[]) => [...new Set(values)]

console.log(uniqueValues(...labels, label))
```